src/components/activity/StatsView.tsx

```diff
@@ -90,6 +90,8 @@ const StatsView = () => {
   const [personalTicketStats, setPersonalTicketStats] = useState<TicketStats[]>([]);
   const [globalTimeRangeOption, setGlobalTimeRangeOption] = useState(timeRangeOptions[0]);
   const [personalTimeRangeOption, setPersonalTimeRangeOption] = useState(timeRangeOptions[0]);
+  const [globalStartDate, setGlobalStartDate] = useState<Date>();
+  const [personalStartDate, setPersonalStartDate] = useState<Date>();
 
   const { isLoading: isStatsLoading } = trpc.stats.getTicketStats.useQuery(undefined, {
```

Comment:
I think the issue is that this call i still running right? This query sill exceeds the 4MB limit
![image](https://user-images.githubusercontent.com/24885081/232155289-ea9d9347-644f-4c95-bca7-d8a980850e07.png)

Reply:
oh this was noah's other request, im still trying to figure out the 4mb thing